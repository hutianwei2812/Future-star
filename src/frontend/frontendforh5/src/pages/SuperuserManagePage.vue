<template>
  <div class="superusermanagepage">
    <title-bar :is-on-load="isOnLoad"></title-bar>
    <nav-bar :special-color="specialColor"></nav-bar>
    <br />
    <h1 class="h1Style">
      <font class="fontStyle" color="grey">您拥有超级管理员身份</font>
    </h1>
    <h2 class="h2Style">
      <font class="fontStyle" color="grey">有权执行下列操作</font>
    </h2>
    <SuperUserManageTable />
  </div>
</template>

<script>
import TitleBar from '@/components/TitleBar.vue'
import NavBar from '@/components/NavBar.vue'
import SuperUserManageTable from '@/components/SuperUserManageTable.vue'
export default {
  name: 'SuperUserManagePage',
  data() {
    return {
      isOnLoad: true,
      specialColor: 5
    }
  },
  components: {
    SuperUserManageTable,
    TitleBar,
    NavBar
  }
}
</script>

<style scoped type="type/css">
.h1Style {
  text-align: center;
  font-weight: normal;
}
.fontStyle {
  font-family: '微软雅黑';
}
.h2Style {
  text-align: center;
  font-weight: normal;
}
</style>