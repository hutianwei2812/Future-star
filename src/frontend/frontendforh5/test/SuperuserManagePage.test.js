import { mount } from '@vue/test-utils'
import SuperuserManagePage from '@/pages/SuperuserManagePage'
import Vue from 'vue'
import ElementUI from 'element-ui'
Vue.use(ElementUI)

test('renders correctly', () => {
  const wrapper = mount(SuperuserManagePage)
  expect(wrapper.element).toMatchSnapshot()
})