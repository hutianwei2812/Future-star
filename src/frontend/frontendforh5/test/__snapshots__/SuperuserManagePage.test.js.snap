// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="superusermanagepage"
>
  <header
    class="el-header"
    style="height: 60px;"
  >
    <span
      class="title-span"
    >
      未来之星管理系统
    </span>
     
    <span
      class="user-span"
    >
      
    </span>
     
    <a
      class="el-link el-link--primary is-underline"
    >
      <!---->
      <span
        class="el-link--inner"
      >
        注销
      </span>
      <!---->
    </a>
  </header>
   
  <div
    class="nav"
  >
    <section
      class="el-container is-vertical"
    >
      <header
        class="el-header"
        style="height: 60px;"
      >
        <div
          class="newDiv"
        >
          <ul
            class="sub_nav"
          >
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                报名管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                课程管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                活动管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                校友管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="blueColor"
              >
                管理员权限设置
              </li>
            </div>
          </ul>
        </div>
      </header>
    </section>
  </div>
   
  <br />
   
  <h1
    class="h1Style"
  >
    <font
      class="fontStyle"
    >
      
    您拥有超级管理员身份
    </font>
  </h1>
   
  <h2
    class="h2Style"
  >
    
    有权执行下列操作
  
  </h2>
   
  <div
    class="managetable"
  >
    <div
      class="NoAllow"
    >
      <ul
        class="th"
      >
        <li>
          管理员账号
        </li>
         
        <li>
          管理员密码
        </li>
         
        <li>
          新建创新院权限
        </li>
         
        <li>
          课程管理权限
        </li>
         
        <li>
          新建活动权限
        </li>
         
        <li>
          新建管理员权限
        </li>
         
        <li>
          暂不可操作
        </li>
         
        <!---->
      </ul>
       
    </div>
     
    <br />
     
    <button
      class="el-button NewButton el-button--primary"
      type="button"
    >
      <!---->
      <i
        class="el-icon-edit"
      />
      <span>
        编 辑
      </span>
    </button>
     
    <button
      class="el-button InsertButton el-button--primary"
      type="button"
    >
      <!---->
      <i
        class="el-icon-plus"
      />
      <span>
        新建管理员
      </span>
    </button>
     
    <!---->
     
    <br />
     
    <br />
     
    <div
      class="el-dialog__wrapper"
      name="dialog-fade"
      style="display: none;"
    >
      <div
        aria-label="提示"
        aria-modal="true"
        class="el-dialog"
        role="dialog"
        style="margin-top: 15vh; width: 30%;"
      >
        <div
          class="el-dialog__header"
        >
          <span
            class="el-dialog__title"
          >
            提示
          </span>
          <button
            aria-label="Close"
            class="el-dialog__headerbtn"
            type="button"
          >
            <i
              class="el-dialog__close el-icon el-icon-close"
            />
          </button>
        </div>
        <!---->
        <div
          class="el-dialog__footer"
        >
          <span
            class="dialog-footer"
          >
            <button
              class="el-button el-button--default"
              type="button"
            >
              <!---->
              <!---->
              <span>
                取 消
              </span>
            </button>
             
            <button
              class="el-button el-button--primary"
              type="button"
            >
              <!---->
              <!---->
              <span>
                确 定
              </span>
            </button>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
